checker/src/org/checkerframework/checker/index/upperbound/UBQualifier.java

```diff
@@ -287,14 +289,39 @@ public boolean isLessThanLengthOf(String sequence) {
          * AnnotationMirrors using @{@link LTEqLengthOf} or @{@link LTOMLengthOf} are returned.
          * Otherwise, @{@link LTLengthOf} is used.
          *
-         * <p>The annotation is sorted by sequence and then offset. This is so that {@link
-         * AnnotationUtils#areSame(AnnotationMirror, AnnotationMirror)} returns true for equivalent
-         * annotations.
+         * <p>The returned annotation is canonicalized by sorting its arguments by sequence and then
+         * offset. This is so that {@link AnnotationUtils#areSame(AnnotationMirror,
+         * AnnotationMirror)} returns true for equivalent annotations.
          *
          * @param env ProcessingEnvironment
          * @return the AnnotationMirror that represents this qualifier
          */
-        public AnnotationMirror convertToAnnotationMirror(ProcessingEnvironment env) {
+        public AnnotationMirror convertToAnnotation(ProcessingEnvironment env) {
+            return convertToAnnotation(env, false);
+        }
+
+        /**
+         * Returns the @{@link SubstringIndexFor} AnnotationMirror that represents this qualifier in
+         * the Substring Index hierarchy.
+         *
+         * @param env ProcessingEnvironment
```

Comment:
This documentation is not informative (same comment for other methods in the class where the comment is just the type).